spaceship/lib/spaceship/two_step_client.rb

```diff
@@ -35,7 +35,7 @@ def handle_two_step(response)
         result = choose(*available)
         device_id = result.match(/.*\t.*\t\((.*)\)/)[1]
         select_device(r, device_id)
-      elsif r.body.kind_of?(Hash) && r.body["phoneNumberVerification"].kind_of?(Hash)
+      elsif r.body.kind_of?(Hash) && r.body["trustedPhoneNumbers"].kind_of?(Array) && r.body["trustedPhoneNumbers"].first.kind_of?(Hash)
```

Comment:
We still use `phoneNumberVerification` in the `handle_two_factor` method, shouldn't we check for this?